src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6866,9 +6883,13 @@ private ErrorPropertySymbol CreateErrorPropertySymbol(ImmutableArray<PropertySym
             AnalyzedArguments analyzedArguments,
             bool isMethodGroupConversion,
             ref HashSet<DiagnosticInfo> useSiteDiagnostics,
-            bool inferWithDynamic = false)
+            bool inferWithDynamic = false,
+            RefKind returnRefKind = default,
```

Comment:
I assume you reviewed all the callers. Optional parameters make me nervous, as the PR will not highlight all the call locations which need reviewing.
If most callers had to be updated anyways, consider making the parameters non-optional.

Reply:
I will add comments. These defaults are correct for almost all callers. They are only used when isMethodGroupConversion is true and we are not inferring the return type.

---
In reply to: [167680751](https://github.com/dotnet/roslyn/pull/24756#discussion_r167680751) [](ancestors = 167680751)